Classes/Views/CodeTextView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CodeTextView.swift
+//  Freetime
+//
+//  Created by Weyert de Boer on 12/10/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+final class CodeTextView: UIView, UITextViewDelegate {
```

Comment:
I'm liking where this is going, be great to see it finished!

Reply:
Going to be my attempt to get line numbers. If you know a good approach don't hesitate to let me know!